src/iop/temperature.c

```diff
@@ -1622,6 +1661,7 @@ void reload_defaults(dt_iop_module_t *module)
     dt_bauhaus_combobox_add(g->presets, C_("white balance", "user modified"));
     // old "camera neutral", reason: better matches intent
     dt_bauhaus_combobox_add(g->presets, C_("white balance", "camera reference"));
+    dt_bauhaus_combobox_add(g->presets, C_("white balance", "late camera reference"));
```

Comment:
We need more comment about this as I'm not sure what it represents exactly and will have trouble during translation to French. Maybe a better wording for this new option is possible?

Reply:
The tooltip added below is:
`set white balance to as shot and later correct to camera reference point`
But then what is the difference between `camera reference` and `late camera reference`?

Reply:
I agree and the wording is just "preliminary". I am not even sure we want the extra button at all. From my understanding - and if all reviewers and testers agree on this - i would 
1. keep old behaviour for current image developments
2. use new algo by selecting the reset to default
3. use new algo for new edits.

But all are open for discussion.